packages/sirius-web/backend/sirius-web/src/test/java/org/eclipse/sirius/web/application/controllers/libraries/LibraryControllerIntegrationTests.java

```diff
@@ -40,6 +56,25 @@ public class LibraryControllerIntegrationTests extends AbstractIntegrationTests
     @Autowired
     private LibrariesQueryRunner librariesQueryRunner;
 
+    @Autowired
+    private PublishLibrariesMutationRunner publishLibrariesMutationRunner;
+
+    @Autowired
+    private ILibrarySearchService librarySearchService;
+
+    @Autowired
+    private IStandardLibrarySemanticDataInitializer standardLibrarySemanticDataInitializer;
+
+    @BeforeEach
+    public void beforeEach() {
+        // Re-create the Papaya standard library before each test: it will be deleted by the cleanup script.
+        var initializeJavaStandardLibraryEvent = new InitializeStandardLibraryEvent(UUID.randomUUID(), "java", "Java Standard Library", "17.0.0", "The standard library of the Java programming language");
+        this.standardLibrarySemanticDataInitializer.initializeStandardLibrary(initializeJavaStandardLibraryEvent);
+        TestTransaction.flagForCommit();
+        TestTransaction.end();
+        TestTransaction.start();
```

Comment:
I have to commit the transaction to trigger the `@TransactionalEventListener` that creates the library. It feels weird though

Reply:
I'll see if we can't just move it in the sql scripts, it will be available for all tests this way. It will help create more complex tests.